src/main/java/com/minecolonies/coremod/colony/StructureName.java

```diff
@@ -101,7 +104,7 @@ else if (name.startsWith(Structures.SCHEMATICS_CACHE + Structures.SCHEMATICS_SEP
             hut = schematic.split("\\d+")[0];
             section = Structures.SCHEMATICS_PREFIX;
 
-            if (Block.getBlockFromName(Constants.MOD_ID + ":blockHut" + hut) != null)
+            if (Block.getBlockFromName(Constants.MOD_ID + HUT_PREFIX + hut) != null || Block.getBlockFromName(Constants.MOD_ID + (HUT_PREFIX + hut).toLowerCase(Locale.ENGLISH)) != null)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 185 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)